src/entity/Source.ts

```diff
@@ -22,6 +22,19 @@ export enum SourceType {
   Squad = 'squad',
 }
 
+interface SourceFlags {
+  featured: boolean;
+  totalViews: number;
+  totalPosts: number;
+  totalUpvotes: number;
+}
```

Comment:
Also please follow `SourceFlagsPublic` nomenclature, you can see example for  `UserPostEntity`. Basically public flags are those that can be requested/updated from graphql.